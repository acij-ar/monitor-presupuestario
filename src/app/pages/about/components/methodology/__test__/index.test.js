const React = require('react');
const { shallow } = require('enzyme');

const AboutMethodology = require('..');

describe('AboutMethodology component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<AboutMethodology/>);
    expect(wrapper).toMatchSnapshot();
  });
});
