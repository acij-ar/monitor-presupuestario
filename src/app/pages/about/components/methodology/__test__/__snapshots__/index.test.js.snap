// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMethodology component should match snapshot 1`] = `
<ContentSection
  id="about-methodology"
  title="Metodología"
>
  <p>
    Para desarrollar el Monitor presupuestario utilizamos la información provista por el Ministerio de Hacienda, específicamente los dataset de Crédito y Gasto completo (distribución diaria acumulada de los créditos presupuestarios con clasificadores completos) desde el año 2007 al año 2018, disponible en el sitio presupuestoabierto.gob.ar .
  </p>
  <p>
    La información sobre el proyecto de presupuesto 2019 fue solicitada por ACIJ al Ministerio de Hacienda por medio de un pedido de acceso a la información. Asimismo, actualmente se encuentra disponible en el sitio del Ministerio de Hacienda en 
    <a
      href="https://www.minhacienda.gob.ar/onp/presupuesto_ciudadano/index.html"
      rel="noopener noreferrer"
      target="_blank"
    >
      este link
    </a>
    .
  </p>
</ContentSection>
`;
